common/src/main/java/com/microsoft/identity/common/internal/platform/AndroidPlatformUtil.java

```diff
@@ -206,6 +210,12 @@ public String getPackageNameFromUid(int uid) {
         return mContext.getPackageManager().getNameForUid(uid);
     }
 
+    private boolean isValidRedirectURIForNAA(String redirectUri) {
+        // The only allow-listed hub app on ESTS is Teams app. We cannot use our test app's clientId/redirecrURI for testing NAA scenarios
+        // Below redirectURI is being used in our automation tests and also by OneAuth tests for NAA
+        return redirectUri.equals("msauth://com.microsoft.teams/VCpKgbYCXucoq1mZ4BZPsh5taNE=") || redirectUri.equals("msauth://com.microsoft.teams/fcg80qvoM1YMKJZibjBwQcDfOno=");
```

Comment:
I guess one of them is prod signed and other debug signed?

Reply:
yes